frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -98,6 +98,11 @@
 
         vm.workerLogs = [];
 
+        // get from backend
+        vm.selectedWorkerResources = [0.5, 1024];
+
+        vm.workerResourceOptions = [[0.25, 512], [0.25, 1024], [0.25, 2048], [0.5, 1024], [0.5, 2048], [1, 2048]];
```

Comment:
We should use the actual boto3 values here and just show floating point values of each CPU unit only in dropdown.